packages/tpetra/core/src/Tpetra_CrsMatrix_def.hpp

```diff
@@ -7156,6 +7160,7 @@ namespace Tpetra {
       typedef Kokkos::Device<HES, HostSpace> host_device_type;
       Kokkos::View<const char*, host_device_type>
         exports_a_kv (exports_a.getRawPtr (), newAllocSize);
+      // DEEP_COPY REVIEW - NOT TESTED
```

Comment:
Yes, this case is exporting/importing into a CrsMatrix from a RowMatrix that cannot be cast to CrsMatrix.
It is untested in Tpetra and unexercised in Trilinos.